ETWAnalyzer/EventDump/DumpCPUMethod.cs

```diff
@@ -941,12 +942,18 @@ private void PrintProcessTotalMatches(List<MatchData> matches)
 
         private void WriteCSVProcessTotal(List<MatchData> matches)
         {
-            OpenCSVWithHeader("CSVOptions", "Test Case", "Date", "Test Time in ms", "CPU ms", "Baseline", "Process", "Process Name", "Start Time", "Command Line", "SourceFile", "SourceDirectory", "IsNewProcess", "Module Version");
+            OpenCSVWithHeader("CSVOptions", "Test Case", "Date", "Test Time in ms", "CPU ms", "Baseline", "Process", "Process Name", "Start Time", "Command Line", "SourceFile", "SourceDirectory", "IsNewProcess", "FileVersion", "VersionString", "ProductVersion", "Name", "Description", "ExecutableDirectory"  );
```

Comment:
"Name" should be "ProductName" because it could also be confused with the executable name or some other name in a different context.